t/t6026-merge-attr.sh

```diff
@@ -180,4 +180,28 @@ test_expect_success 'up-to-date merge without common ancestor' '
 	)
 '
 
+cat >./custom-merge <<\EOF
+#$SHELL_PATH
+sleep 1 &
+EOF
+chmod +x ./custom-merge
+
+test_expect_success 'custom merge backend with delay' '
+	git reset --hard anchor &&
+	echo "* merge=union" >.gitattributes &&
+	echo "text merge=custom" >>.gitattributes &&
+
+	git config --replace-all \
+	merge.union.driver "true" &&
+	git config --replace-all \
+	merge.union.name "always ours" &&
+
+	git config --replace-all \
+	merge.custom.driver "./custom-merge %O %A %B 0 %P" &&
```

Comment:
The arguments are ignored by the script. Why should we pass them? Let's not.

Also, do we have to `--replace-all`? If we go with the `sleep-one-second` name, then

```sh
git config merge.sleep-one-second.driver ./sleep-one-second.sh
```
should be enough.